app/experimenter/nimbus-ui/src/components/TableVisualizationRow/index.tsx

```diff
@@ -168,9 +171,11 @@ const TableVisualizationRow: React.FC<{
   const metricData = branch_data[metricKey];
 
   let field = <>{metricName} is not available</>;
+  let tooltipText =
+    metricKey === METRIC.RETENTION ? GENERAL_TIPS.MISSING_RETENTION : "";
   let className = "text-danger";
   if (metricData) {
-    className = "";
+    className = tooltipText = "";
```

Comment:
Does the little `= ""` need to be at the end still?

Reply:
yes. we need to reset it

Reply:
Yeah, it's a double assignment where we're setting *both* `className` and `tooltipText` to an empty string.
I think ESLint has a rule for disallowing multi-assignments and forces you to do two separate `className = ""; tooltipText = "";` assignments instead.